templates/lobby.html

```diff
@@ -739,6 +739,86 @@
     drawingBoard.onmousemove = mouseUpdate;
     drawingBoard.onmouseleave = mouseUpdate;
 
+    let sizeSlot = 0;
+        let toolSlot = 0;
+        let colorSlot = 0;
+        let colorBox = document.getElementsByClassName('color-button');
+        var keyMap = {};
+        document.body.addEventListener('keydown', function(e) {
+            keyMap[e.key] = true;
+        });
+        document.body.addEventListener('keyup', function(e) {
+            keyMap[e.key] = false;
+        });
+        
+        drawingBoard.onwheel = function wheelUpdate(e) {
+            event.preventDefault();
```

Comment:
Shouldn't we only prevent the default action if any of our keys was actually pressed?